sdk/core/core-sse/test/server/responses.mts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Response } from "express";
+import { logger } from "./logger.mts";
+
+export function sendHeaders(res: Response) {
+  res.writeHead(200, {
+    "Cache-Control": "no-cache",
+    "Content-Type": "text/event-stream",
+    Connection: "keep-alive",
+  });
+}
+
+export async function sendEvents(res: Response, count: number, waitTimeInMs: number) {
+  let i = 0;
+  while (i < count) {
```

Comment:
Nit, could this be a for loop?

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/30707/commits/15035295cc9f51ff77d13c28f13eb57daff12ef8